pkg/collector/metric/node_metric.go

```diff
@@ -228,46 +230,46 @@ func (ne *NodeMetrics) ToEstimatorValues(featuresName []string, shouldNormalize
 		case config.GpuUsageMetric: // for GPU resource usage
 			featureValues = append(featureValues, normalize(ne.ResourceUsage[config.GpuUsageMetric], shouldNormalize))
 
-		case PKG + "_DYN": // for dynamic PKG power consumption
+		case PKG + DYN: // for dynamic PKG power consumption
```

Comment:
just curious this is mandatory for the new build process?

Reply:
this was for fixing lint issues. i changed go version to 1.20 and there were lots of lint issues. fixed all reported.
there were many like
```
pkg/collector/metric/node_metric.go:231:14: string `_DYN` has 7 occurrences, make it a constant (goconst)
		case PKG + "_DYN": // for dynamic PKG power consumption
```